benchmarks/solcx/compositional_fields/solcx_particles.prm

```diff
@@ -0,0 +1,133 @@
+# A description of the SolCX benchmark using active particles
```

Comment:
Both solcx_particles.prm and solcx_compositional_fields.prm load the same built library libsolcx_compositional_field.so. This was intentional due to this reason.